src/System.Management.Automation/engine/interpreter/Utilities.cs

```diff
@@ -1096,59 +1096,5 @@ internal static bool TrueForAll<T>(this IEnumerable<T> collection, Predicate<T>
 
             return result;
         }
-
-        // We could probably improve the hashing here
-        internal static int ListHashCode<T>(this IEnumerable<T> list)
```

Comment:
I personally tend to keep these utility code around. It seems useful as reference/sample on collection comparison.

Reply:
Maybe comment the code to exclude from binary?

Reply:
I think we just leave it as is. It's a small chunk of code and thus won't really affect the binary size.